cellprofiler/modules/__init__.py

```diff
@@ -29,7 +29,8 @@
                         'createbatchfiles': 'CreateBatchFiles',
                         'crop': 'Crop',
                         'definegrid': 'DefineGrid',
-                        'dilation': 'Dilation',
```

Comment:
When renaming modules, you'll need to add a line to [substitutions](https://github.com/AetherUnbound/CellProfiler/blob/f6d2f7896154d23b11bede3e8408de81848d31e4/cellprofiler/modules/__init__.py#L215) mapping the old name to the new name.

Reply:
Good to know!